src/ansi-c/scanner.l

```diff
@@ -425,6 +425,20 @@ void ansi_c_scanner_init()
                   PARSER.set_pragma_cprover();
                 }
 
+<CPROVER_PRAGMA>{ws}"check"{ws}"enable"{ws}{named_check} {
```

Comment:
I'm not sure this should be part of the same pull request?

Reply:
I'll clean this up